hbase-server/pom.xml

```diff
@@ -333,6 +333,12 @@
       <artifactId>bcpkix-jdk15on</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.skyscreamer</groupId>
+      <artifactId>jsonassert</artifactId>
+      <version>1.5.1</version>
```

Comment:
Better first add this to the parent pom, in the dependencyManagement section, and then reference it here, without specifying the version, as maybe in the future we also need to use this in other mdules, epsecially the hbase-rest module.